python/tvm/contrib/lldb/tvm.py

```diff
@@ -0,0 +1,201 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
```

Comment:
Given that this is a separate package and cannot be imported separately, let us put it under apps/lldb

Reply:
Thanks. Files have been moved.